templates/cassiopeia/scss/tools/variables/_variables.scss

```diff
@@ -28,7 +28,7 @@ $gray-400:                            hsl(210, 14%, 83%);
 $gray-500:                            hsl(210, 11%, 71%);
 $gray-600:                            hsl(210, 7%, 46%);
 $gray-700:                            hsl(210, 9%, 31%);
-$gray-800:                            hsl(210, 10%, 23%);
+$gray-800:                            rgb(53, 59, 65);
```

Comment:
@drmenzelit why changing from rgb to hsl?
According to https://www.w3schools.com/colors/colors_converter.asp the color is the same

Reply:
@hans2103 You mean "why changing from hsl to rgb?", right? But you wrote "why changing from rgb to hsl?" ;-) I was confused when reading it in my email notification because it was you who wanted the hsl ;-)

Reply:
Changed back to hsl